api/v1alpha1/virtualmachineclass_types.go

```diff
@@ -21,6 +21,30 @@ type DynamicDirectPathIODevice struct {
 	CustomLabel string `json:"customLabel,omitempty"`
 }
 
+// InstanceStorage encapsulates information about instance storage.
+// StorageClass refers to the name of a StorageClass resource used to
+// provide the storage for the configured, instance storage volumes.
+// The value of this field has no relationship to or bearing on the field
+// virtualMachine.spec.storageClass.
+// Please note the referred StorageClass must be available in the same
+// namespace as the VirtualMachineClass that uses it for configuring
+// instance storage.
+type InstanceStorage struct {
+	// StorageClass describes the configuration of instance storage related
+	// attributes on the VirtualMachine instance.
+	StorageClass string `json:"storageClass,omitempty"`
+
+	// Volumes describes a list of InstanceStorageVolumes that are to be
+	// configured on the VirtualMachine. 
+	Volumes []InstanceStorageVolume `json:"volumes"`
+}
+
+// InstanceStorageVolume contains configuration corresponding to an
+// instance storage volume.
+type InstanceStorageVolume struct {
+	Size resource.Quantity `json:"size"`
```

Comment:
We're using `resource.Quantity`, which can be specified in multiple units. Is there some minimum size we should document here? Or is that unimportant since people won't be creating these directly anyway?

Reply:
Size is a required parameter that is not directly specified by the user of VM class. I think the minimum size depends on the type (small, xlarge etc) of VM class.

Reply:
Fair.